repos.d/rpm/openmandriva.yaml

```diff
@@ -0,0 +1,42 @@
+###########################################################################
+# OpenMandriva
+###########################################################################
+- name: openmandriva_3
+  type: repository
+  desc: OpenMandriva 3.x
```

Comment:
Is it not worth to create separate repo for each 3.x? As I see, there's only 3.0 currently, but may 3.1 also appear?